tests/multi_playground.html

```diff
@@ -56,6 +57,7 @@
 
   startBlocklyInstance('HorizontalEndLTR', false, true, 'end', toolbox);
   startBlocklyInstance('HorizontalEndRTL', true, true, 'end', toolbox);
+  Blockly.setStyle(Blockly.Styles.Classic);
```

Comment:
If this isn't set, will the workspace and blocks still load?  Ideally, we want this to be backward compatible with apps that upgrade their Blockly without adding any new code.

Reply:
This was left over from how I was setting the style before so I just deleted it. Currently, i'm setting the style in the inject method. If the user doesn't pass in a style then it will default to the Classic style.